### PR TITLE
DiffEqBio tutorial

### PR DESCRIPTION
**Pull request by isaacsas**
Only html, script and notebook generated due to tex issues with Weave, see [50](https://github.com/JuliaDiffEq/DiffEqTutorials.jl/issues/50). ~No footer due to [48](https://github.com/JuliaDiffEq/DiffEqTutorials.jl/issues/48).~ I can update in the future once these issues are straightened out.

### CODE REVIEW

**Comment by isaacsas** [comment]
I tweaked the footer to say "this tutorial" instead of "these benchmarks", and also added an optional kwarg to swap out a user's `homedir()` with `~` (which I used for the diffeqbio tutorial). I can revert these if either is a problem, but I'd prefer not to have my username in the tutorial (which on my mac is part of the path).

File: src/DiffEqTutorials.jl [hunk 1]
```diff
@@ -57,11 +57,11 @@ function weave_folder(folder)
   end
 end
 
-function tutorial_footer(folder=nothing, file=nothing)
+function tutorial_footer(folder=nothing, file=nothing; remove_homedir=false)
```
Comment (ChrisRackauckas) [inline src/DiffEqTutorials.jl]: we should probably just default this to true.

Comment (isaacsas) [reply src/DiffEqTutorials.jl]: OK, I'll change that.

Comment (isaacsas) [reply src/DiffEqTutorials.jl]: all fixed

**Pull request merged**